site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,250 @@
+- `label`: Controls the wrapping `<label>` element.
+___
+
+### Slider
+A native range slider offers limited visual customization, so the Dojo Slider widget creates an invisible native `<input type="range">` input that controls a custom track, thumb, and fill. This allows the widget to take advantage of native accessibility and keyboard interaction without sacrificing styling.
+
+The current value is wrapped in an `<output>` element that points to the ID of the native input.
+
+##### A11y properties
+- `describedBy`: Sets the `aria-describedby` property to point to the ID of an element containing additional descriptive text. Screen readers usually read this text after the label text, followed by a short pause.
+- `label`: Controls the wrapping `<label>` element.
+___
+
+### Select
+Select is the only Dojo form widget that makes use of a fully custom solution. Since it is impossible to customize `<option>` elements, the component instead re-creates select functionality from scratch. The end result is a popup button that controls a dropdown menu (with `role="listbox"`) of options.
+
+ARIA attributes are used to associate the button with the menu and focused option. They are also used to provide semantics for read-only and disabled options, as well as setting the entire select field to be read-only, invalid, or required.
+
+It is also possible to use Select to create a widget using the native `<select>` element by passing `useNativeElement: true` to the properties object.
+
+##### Keyboard Interaction
+- **Down arrow**: Focuses the next option, wrapping from last to first, or opens the menu if closed
+- **Up arrow**: Focuses the previous option, wrapping from first to last
+- **Home**: Focuses the first option
+- **End**: Focuses the last option
+- **Enter & Space**: Selects the focused option and closes the menu, unless the option is disabled
+- **Escape**: Closes the menu without selecting an option
+
+Disabled options are focusable but not selectable within the menu, per [WAI-ARIA recommendations](https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_disabled_controls).
+
+##### A11y properties
+- `describedBy`: Sets the `aria-describedby` property to point to the ID of an element containing additional descriptive text. Screen readers usually read this text after the label text, followed by a short pause.
+- `label`: Controls the wrapping `<label>` element.
+- `useNativeElement`: Switches to using the native `<select>` element, with some trade-offs to customizability.
+
+___
+
+### ComboBox
+
+ComboBox is similar to Select, but provides a text input that filters available options in addition to a button that opens the option menu. The markup is similar in that the menu comprises of a listbox of options with `role="option"` controlled by both a text input and button.
```

Comment:
Does ComboBox rely on native capabilities then? It might be surprising that it's perhaps less customizable than Select unless we say so?

Reply:
I'm not entirely sure I follow -- is there something in this line that implies Combobox uses native capabilities or is less customizable? The properties interface is quite different, but not necessarily in a way that directly relates to accessibility.

Reply:
You might think of a ComboBox as a Select with the ability to type in a custom value. Previously we mention that Select is the only widget where we do a fully custom implementation by default so you can style the options. So someone reading this will wonder if ComboBox is like Select in that approach, or if it's limited to whatever the native select and option tags support.

Reply:
Ah, I think it was the only form field widget with a fully custom implementation. I wasn't really thinking of ComboBox as a form widget, but I suppose it kind of is. I'll change the wording under `Select`